File: Leet_Code/Problem_12/Code.py
class Solution:
    def reverseWords(self, s: str) -> str:
        res = ""
        index = 0
        for element in s.split(" "):
            if index == len(s.split(" ")) - 1:
                res += element[::-1]
            else:
                res += element[::-1] + " "
            index += 1
        return res
        
        

File: Leet_Code/Problem_34/Code.js
/**
 * @param {string} s1
 * @param {string} s2
 * @return {string[]}
 */
 var uncommonFromSentences = function(s1, s2) {
    const res = (s1, s2) => s1.concat(' ',s2)
    .split(' ')
    .filter((e, index, arr) => arr.indexOf(e) === arr.lastIndexOf(e))
    return res(s1, s2)

};